includes/custom-templates.md

```diff
@@ -246,10 +244,37 @@ Declare **named** templates within the component's markup but outside the templa
     })
     export class AppModule { }
 
-##### Vue
-
 Declare **named** templates within the component's markup but outside the templated element. **Non-named** templates should be declared inside the templated element.
 
+    <!-- tab: app.component.html -->
+    <!-- Named template (the name is 'cellTemplate'): -->
+    <dx-data-grid ... >
+        <dxi-column
+            dataField="Picture"
+            [width]="100"
+            [allowFiltering]="false"
+            [allowSorting]="false"
+            cellTemplate="cellTemplate"
```

Comment:
Think its better to use another name
Like here
https://js.devexpress.com/Angular/Demos/WidgetsGallery/Demo/DataGrid/CellCustomization/MaterialBlueLight/